draft-ietf-mls-architecture.md

```diff
@@ -1524,7 +1528,7 @@ properties, but defining the specific requirements and resulting notions of
 deniability requires further analysis.
 
 
-### Associating a User's Clients
+### Associating a User's Clients {#associating-a-users-clients}
```

Comment:
This tag shouldn't be necessary; kramdown auto-generates tags of this form based on the text of the heading.

Reply:
I was worried about the ' but yeah, it seems fine.